seatunnel-connectors-v2/connector-doris/src/main/java/org/apache/seatunnel/connectors/doris/util/DorisCatalogUtil.java

```diff
@@ -153,6 +162,12 @@ public static String getCreateTableStatement(
                         rowTypeFields);
     }
 
+    private static boolean canHandledByDefaultTemplate(String createTemplate, String primaryKey) {
+        return StringUtils.equals(
+                        createTemplate, DorisOptions.SAVE_MODE_CREATE_TEMPLATE.defaultValue())
+                && StringUtils.isBlank(primaryKey);
+    }
```

Comment:
I thought about it again, whether this kind of check is too rigid, and its useful scenarios are very limited. We can predict whether the spliced SQL will be abnormal by checking whether the placeholders and corresponding values used in the template exist. For example, by default template, we know that the primarykeys information needs to be used by scanning the template, but the primarykeys information is not found, then we can throw an exception.
```java
    private static boolean canHandledByDefaultTemplate(String createTemplate, String primaryKey) {
        if (createTemplate.contains(String.format("\\$\\{%s\\}", SaveModeConstants.ROWTYPE_PRIMARY_KEY)) && StringUtils.isEmpty(primaryKey)) {
            throw exception;
        }
        .... same for uniqueKeys
    }
```

Reply:
Good idea.

Reply:
My intention is to do so, because the relationship of time I made the simplest. I'm going to take some time to do that.